src/app/campaign/campaign-form.component.html

```diff
@@ -0,0 +1,54 @@
+<form [formGroup]="campaignForm" (ngSubmit)="campaignFormSubmit()" ngNativeValidate>
+  <label>
+    Owner:
+    <select formControlName="accountId" required>
```

Comment:
This may be a more recent change to Augury ... but you'll need to do `set_account_uri` here.  (I disabled setting account-ids directly, in favor of the more HAL-ish approach).